autopush/web/registration.py

```diff
@@ -71,7 +74,19 @@ def convert_chid(self, data):
 class TokenSchema(SubInfoSchema):
     """Filters allowed values from body data"""
     token = fields.Str(allow_none=True)
-    # Temporarily allow 'aps' definition data for iOS.
+
+
+def validate_token(val):
```

Comment:
Huh, didn't know about the RegExp validator, might be useful. Tokens should be strings, but again, are opaque to us. (Granted, I don't see google or apple going to making these transparent, complex structures for a lot of reasons.)